requirements/extras/zstd.txt

```diff
@@ -1 +1 @@
-zstandard
+zstandard ; python_version < "3.13"
```

Comment:
This line is necessary for the installation of requirements. The change should be removed before the merge.

https://github.com/indygreg/python-zstandard/pull/233

Reply:
https://github.com/indygreg/python-zstandard/issues/210#issuecomment-2336604856